src/App.test.js

```diff
@@ -1,91 +1,97 @@
 import React from 'react';
 import '@testing-library/jest-dom/extend-expect';
 import App from './App';
-import { render, screen, fireEvent, waitFor, prettyDOM} from '@testing-library/react'
-
+import {
+  render,
+  screen,
+  fireEvent,
+  waitFor,
+  prettyDOM,
+} from '@testing-library/react';
 
 describe('App', () => {
   // Helper function to click on buttons in the grid
-  const clickButtonAndVerifyResult = (container, buttonIndex, expectedResult) => {
+  const clickButtonAndVerifyResult = (
+    container,
+    buttonIndex,
+    expectedResult
+  ) => {
     let buttons = container.querySelectorAll('.grid button');
     fireEvent.click(buttons[buttonIndex]);
-    
+
     buttons = container.querySelectorAll('.grid button');
     expect(buttons[buttonIndex].innerHTML).toEqual(expectedResult);
-  }
-
-  describe.skip('Wave 2: clicking on squares and rendering App', () => {
```

Comment:
There were also skipped tests in Board.test.js and Square.test.js. They passed when I enabled them!